runtime/doc/options.txt

```diff
@@ -644,15 +644,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	escape sequence to request cursor position report.
 
 			*'antialias'* *'anti'* *'noantialias'* *'noanti'*
-'antialias' 'anti'	boolean (default: off)
-			global
-			{only available when compiled with GUI enabled
-			on Mac OS X}
-	This option only has an effect in the GUI version of Vim on Mac OS X
-	v10.2 or later.  When on, Vim will use smooth ("antialiased") fonts,
-	which can be easier to read at certain sizes on certain displays.
-	Setting this option can sometimes cause problems if 'guifont' is set
-	to its default (empty string).
+'antialias' 'anti'	Removed. {Nvim}
```

Comment:
I think that if you completely remove this option it should be removed from here completely as well, but a note should be added to `vim_diff.txt`.